pkg/awxrunner/awx_action_runner.go

```diff
@@ -167,3 +206,7 @@ func (h *Healer) checkAWXJobStatus(jobID int) (finished bool, err error) {
 
 	return
 }
+
+func (r *AWXActionRunner) AWX() *config.AWXConfig {
```

Comment:
I still think that this method isn't needed, and using it hides the fact that we are using the configuration. Can you remove this method and use directly `config...`?

Reply:
Yeah I forgot to remove it :cry: